src/newlines.cpp

```diff
@@ -3737,11 +3737,20 @@ void newlines_cleanup_braces(bool first)
          }
       }
       else if (  chunk_is_token(pc, CT_NAMESPACE)
-              && pc->parent_type != CT_USING)
+              && pc->parent_type != CT_USING
+              && pc->next
```

Comment:
I prefer pc->next != nullptr

Reply:
Done. I also tweaked the code to be a bit more robust, e.g. in the face of pathological code with comments / newlines in the middle of a namespace alias. (I can't imagine such code being common, but...)

Reply:
Fine